### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by bgruening**
is that working?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -87,17 +105,15 @@ RUN julia --eval 'using Pkg; pkg"precompile"'
...
 USER root
 # Example Data
-RUN mkdir /data
-RUN mkdir /data/Diva-Workshops-data
+RUN mkdir -p /data/Diva-Workshops-data && \
+    chown jovyan:jovyan /data -R
+USER jovyan
+
 RUN curl https://dox.ulg.ac.be/index.php/s/Px6r7MPlpXAePB2/download | tar -C /data/Diva-Workshops-data -zxf -
 RUN ln -s /opt/julia-* /opt/julia
```
Comment (bgruening) [inline Dockerfile]: @Marie59 what is the intension here? We are installing Julia with conda to make the notebook work, but here you are downloading Julia and use a different version that we installed previously.

Julia is available in version 1.8 and 1.9 https://anaconda.org/conda-forge/julia

Comment (Marie59) [reply Dockerfile]: Done !

**Comment by bgruening** [comment]
@Marie59 I have reworked the entire Dockerfile. I have no clue what I'm doing. But the idea is that we use Julia pre-installed. We do not care about Julia, just about the Divand extension, which we compile, and the example files.

I think this now at least compiles. But I do not know if it also works. If you want to take it over from here, feel free, bedtime.

**Comment by Marie59** [comment]
Okay Thank you !! I will see the outcome of that and see what else needs to be done.

**Comment by Marie59** [comment]
> @Marie59 I have reworked the entire Dockerfile. I have no clue what I'm doing. But the idea is that we use Julia pre-installed. We do not care about Julia, just about the Divand extension, which we compile, and the example files.
> 
> I think this now at least compiles. But I do not know if it also works. If you want to take it over from here, feel free, bedtime.

I tried this docker locally and everything seems to work fine for me.

**Pull request merged**